src/providers/bitpay-card/bitpay-card.ts

```diff
@@ -490,14 +494,39 @@ export class BitPayCardProvider {
     });
   }
 
-  public register() {
+  setBitpayCardBannerStatus(isBannerVisible): Promise<void> {
+    return this.persistenceProvider.setBitPayCardBanner(isBannerVisible);
+  }
+
+  getBitpayCardBannerStatus() {
+    return this.persistenceProvider.getBitPayCardBannerStatus();
+  }
+
+  public async register() {
+    const isBannerVisible = await this.persistenceProvider.getBitPayCardBannerStatus();
+    const showIntegration = !!this.configProvider.get().showIntegration[
+      'debitcard'
+    ];
+
+    let isBannerStatusUndefined = _.isNil(isBannerVisible);
+    let countryCode;
+
+    try {
+      const res = await this.globalization.getLocaleName();
+      countryCode = res.value.split('-')[1];
+    } catch (error) {
+      this.logger.warn('Error retrieving country Code', error);
+    }
     this.isActive(isActive => {
       this.homeIntegrationsProvider.register({
         name: 'debitcard',
         title: 'BitPay Visa® Card',
         icon: 'assets/img/bitpay-card/icon-bitpay.svg',
         page: 'BitPayCardIntroPage',
-        show: !!this.configProvider.get().showIntegration['debitcard'],
+        show: isBannerStatusUndefined
+          ? showIntegration &&
+            (countryCode === 'US' || !this.platformProvider.isCordova)
```

Comment:
For fresh install it isn't working (any location: US or AR). Banner is hidden by default.